templates/master/01-master-kubelet/baremetal/units/kubelet.yaml

```diff
@@ -1,36 +0,0 @@
-name: "kubelet.service"
```

Comment:
isn't this undoing the remove-noschedule-taint part of openshift/machine-config-operator#846 ?

Reply:
I don't think that's supposed to be here. Not really certain how that snuck in. Good catch. Thanks @markmc